pkg/cmd/util/clientcmd/factory.go

```diff
@@ -234,17 +238,19 @@ func NewFactory(clientConfig kclientcmd.ClientConfig) *Factory {
 	}
 	kLogsForObjectFunc := w.Factory.LogsForObject
 	w.LogsForObject = func(object, options runtime.Object) (*kclient.Request, error) {
-		oc, _, err := w.Clients()
```

Comment:
I wanted to but didn't know how to convert. I'll try that in a bit. Thanks.

On Monday, November 23, 2015, David Eads <notifications@github.com> wrote:

> In pkg/cmd/util/clientcmd/factory.go
> <https://github.com/openshift/origin/pull/5996#discussion_r45601376>:
>
> > @@ -234,17 +238,19 @@ func NewFactory(clientConfig kclientcmd.ClientConfig) *Factory {
> >  	}
> >  	kLogsForObjectFunc := w.Factory.LogsForObject
> >  	w.LogsForObject = func(object, options runtime.Object) (*kclient.Request, error) {
> > -		oc, _, err := w.Clients()
>
> Why not use a similar check for OriginKind? I'd be willing accept the
> reflective cost. api.Scheme.ObjectVersionAndKind I think.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5996/files#r45601376>.
>